src/app/+item-page/edit-item-page/item-private/item-private.component.spec.ts

```diff
@@ -59,6 +63,19 @@ describe('ItemPrivateComponent', () => {
 
     notificationsServiceStub = new NotificationsServiceStub();
 
+    objectUpdatesServiceStub = {
+      initialize: () => {
+        // do nothing
+      },
+    };
+
+    relationshipService = jasmine.createSpyObj('relationshipService',
+      {
+        getItemRelationshipTypesArray: observableOf([]),
+        getItemRelationshipsArray: observableOf([]),
+      }
+    );
```

Comment:
I'm not sure I understand why these `objectUpdatesServiceStub` and `relationshipService` blocks of code were added to *all* specs under the `edit-item-page`.  It seems odd to have to initialize these to empty in a bunch of unrelated specs.  I see this same change in `item-public.component.spec.ts`, `item-reinstate.component.spec.ts`, `item-withdraw.component.spec.ts`, `abstract-simple-item-action.component.spec.ts` and `abstract-simple-item-action.component.ts`

Reply:
This was because these services were added in the common AbstractSimpleItemActionComponent constructor. I changed that now, the ItemDeleteComponent related services are now added in the ItemDeleteComponent specific constructor, so only that spec file has to be changed.